main.go

```diff
@@ -35,11 +35,10 @@ import (
 	"time"
 )
 
-const volMount = "/git"
-
 var flRepo = flag.String("repo", envString("GIT_SYNC_REPO", ""), "git repo url")
 var flBranch = flag.String("branch", envString("GIT_SYNC_BRANCH", "master"), "git branch")
 var flRev = flag.String("rev", envString("GIT_SYNC_REV", "HEAD"), "git rev")
+var flRoot = flag.String("root", envString("GIT_SYNC_ROOT", "/git"), "root directory for git operations")
```

Comment:
It makes testing manually much easier.

On Tue, Sep 13, 2016 at 4:54 PM, Mike Danese <notifications@github.com>
wrote:

> In main.go
> <https://github.com/kubernetes/git-sync/pull/2#discussion_r78666268>:
>
> >  var flRepo = flag.String("repo", envString("GIT_SYNC_REPO", ""), "git repo url")
> >  var flBranch = flag.String("branch", envString("GIT_SYNC_BRANCH", "master"), "git branch")
> >  var flRev = flag.String("rev", envString("GIT_SYNC_REV", "HEAD"), "git rev")
> > +var flRoot = flag.String("root", envString("GIT_SYNC_ROOT", "/git"), "root directory for git operations")
>
> This seems like an implementation detail. Is this actually useful to
> parametrize?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/git-sync/pull/2/files/bef13a4d4f24aad8eefe85d320633a984994be6f..7c41b1768578f8e5632ce1c7c75362459811e2e4#r78666268>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVITpuNnaVbZKBYof6xYCYoqNhEhYks5qpzfCgaJpZM4Jssne>
> .
>